blaze-core/src/test/scala/org/http4s/blaze/util/EntityBodyWriterSpec.scala

```diff
@@ -232,34 +228,30 @@ class EntityBodyWriterSpec extends Http4sSpec {
 
     "write a deflated resource" in {
       val p = resource through deflate()
-      p.runLog.map(_.toArray) must returnValue(DumpingWriter.dump(p))
+      p.runLog.map(_.toArray) must returnValue(DumpingWriter.dump(resource through deflate()))
     }
 
     "must be stack safe" in {
-      val p = repeatEval(Task.async[Byte]{ _(Right(0.toByte))}(Strategy.sequential)).take(300000)
```

Comment:
Sequential was used here to stress the stack safety.  A direct ExecutionContext (no trampoline, no thread pool) would be the equivalent.